packages/ingest/test/url-path-template.js

```diff
@@ -1,8 +1,26 @@
 'use strict';
 
 const test = require('ava');
+const fs = require('fs');
+const { parseString } = require('xml2js');
+const { xmlParseOptions } = require('@cumulus/cmrjs/utils');
 const urlPathTemplate = require('../url-path-template');
 
+// eslint-disable-next-line max-len
+const modisXmlFile = '../../packages/test-data/granules/MOD09GQ.A2016358.h13v04.006.2016360104606.cmr.xml';
```

Comment:
You should be able to have the test data package as an ingest package dependency in the package.json and use it that way instead of doing a relative path.

Reply:
I tried the following and it didn't work and couldn't import the xml file as a module.
`const modisXmlFile = require('@cumulus/test-data/granules/MOD09GQ.A2016358.h13v04.006.2016360104606.cmr.xml'`

Reply:
One option would be putting @cumulus/test-data in the dependencies, then doing something like this:

```js
const modisXmlFile = path.join(__dirname, '..', 'node_modules/@cumulus/test-data/granules/MOD09GQ.A2016358.h13v04.006.2016360104606.cmr.xml';
```

Probably not super necessary, but using the package from node_modules would help by making sure it's using the version of the test-data package we expect.

Reply:
What error are you getting when you try to get it as a module?